tikv-configuration-file.md

```diff
@@ -1539,6 +1539,16 @@ Configuration items related to `rocksdb.defaultcf`, `rocksdb.writecf`, and `rock
     - `5`: Can be read by TiKV v6.1 and later versions. Full and partitioned filters use a faster and more accurate Bloom filter implementation with a different schema.
 + Default value: `2`
 
+### `ttl` <span class="version-mark">New in v7.2.0</span>
++ SST files containing updates older than TTL will go through the compaction process. This usually happens in a cascading way so that those entries will be compacted to bottommost level/file.
```

Comment:
```suggestion
+ SST files with updates older than the TTL will be automatically selected for compaction. These SST files will go through the compaction in a cascading way so that they can be compacted to the bottommost level or file.
```